pkg/vm/airdrop.go

```diff
@@ -0,0 +1,78 @@
+package vm
+
+import (
+	"errors"
+	"math/big"
+
+	"github.com/ava-labs/avalanche-cli/cmd/prompts"
+	"github.com/ava-labs/subnet-evm/core"
+)
+
+const stageAfterAirdrop = precompileStage
```

Comment:
As all stages are defined in `evm_settings.go`, it seems clearer to me to add these special stages there too, in order to avoid scattering these all over.

I understand your thinking, you're wanting to see them here in this file because these special ones are (probably) used just here. 

Nevertheless I vote for a central place.

Reply:
Finally, even if it seems a good idea to have "labels" on existing stages to help understand the current stage, I think it can quickly become confusing, and make refactoring complex and error prone if needed.

If possible, I would like to suggest to avoid labeling stages and just use the same stage name where the stage applied. Maybe different stage names could help.

Don't get me wrong, I love state machines :D

Reply:
I removed these labels and replaced them with a state direction/incrementing strategy